### DISCUSSION TITLE
Using saver in pyscript

### DISCUSSION THREAD

**Discussion by alex-tomov** [PiotrMachowski/Home-Assistant-custom-components-Saver#18] [Show and tell]
In case anyone is wondering, here is how I managed to save json array of values between calls to my script in pyscript.

`saver.set_variable(name='test_var', value=‘[{“key”:”some_value”}]’)`

And here is how I retrieved it:

```
import json
a=json.loads(saver.saver.variables['test_var'])
a[0][‘key’]
```
I needed to implement some REST Api calls in a queue. The provider had "calls per minute" limit and I saved a history of call timestamps to calculate the next allowed call time.

**Comment by PiotrMachowski** [comment]
Nice 👍
